src/Sarif/Autogenerated/VersionControlDetails.cs

```diff
@@ -10,6 +10,9 @@
 
 namespace Microsoft.CodeAnalysis.Sarif
 {
+    /// <summary>
+    /// Specifies the information necessary to retrieve a desired revision from a version control system.
```

Comment:
>Specifies [](start = 8, length = 9)

Occasionally we see one of these comments come in late. Presumably somebody failed to add the autogenerated file to the PR the last time they changed this string.